Plugin io.fabric8:fabric8-maven-plugin:3.1.74 or one of its dependencies could not be resolved

Jenkins just stopped building my applications with this output:

```
mvn package -U io.fabric8:fabric8-maven-plugin:3.1.74:build
[INFO] Scanning for projects...
[INFO] Downloading: http://nexus/content/groups/public/io/fabric8/fabric8-maven-plugin/3.1.74/fabric8-maven-plugin-3.1.74.pom
[WARNING] The POM for io.fabric8:fabric8-maven-plugin:jar:3.1.74 is missing, no dependency information available
[INFO] Downloading: http://nexus/content/groups/public/io/fabric8/fabric8-maven-plugin/3.1.74/fabric8-maven-plugin-3.1.74.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.856 s
[INFO] Finished at: 2016-10-17T08:09:23+00:00
[INFO] Final Memory: 14M/243M
[INFO] ------------------------------------------------------------------------
[ERROR] Plugin io.fabric8:fabric8-maven-plugin:3.1.74 or one of its dependencies could not be resolved: Could not find artifact io.fabric8:fabric8-maven-plugin:jar:3.1.74 in nexus (http://nexus/content/groups/public) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginResolutionException
sh-4.2# exit
exit
```

When I go to for example https://mvnrepository.com/artifact/io.fabric8/fabric8-maven-plugin the latest release is 3.1.73.
Could you please either publish 3.1.74 or change back to 3.1.73?

--- a/vars/mavenCanaryRelease.groovy
+++ b/vars/mavenCanaryRelease.groovy
@@ -7,12 +7,10 @@ def call(body) {
     body()
 
     def flow = new io.fabric8.Fabric8Commands()
-    def fabric8MavenPluginVersion = flow.getReleaseVersion "io/fabric8/fabric8-maven-plugin"
 
     sh "git checkout -b ${env.JOB_NAME}-${config.version}"
     sh "mvn org.codehaus.mojo:versions-maven-plugin:2.2:set -U -DnewVersion=${config.version}"
-    sh "mvn clean install"
-    sh "mvn install -U io.fabric8:fabric8-maven-plugin:${fabric8MavenPluginVersion}:build"
+    sh "mvn clean -U install"
     
     def s2iMode = flow.isOpenShiftS2I()
     echo "s2i mode: ${s2iMode}"
@@ -24,12 +22,13 @@ def call(body) {
         def user = groupId[groupId.size()-1].trim()
         def artifactId = m.artifactId
 
-       if (!s2iMode) {   kubernetes.image().withName("${user}/${artifactId}:${config.version}").tag().inRepository("${env.FABRIC8_DOCKER_REGISTRY_SERVICE_HOST}:${env.FABRIC8_DOCKER_REGISTRY_SERVICE_PORT}/${user}/${artifactId}").withTag("${config.version}")
+       if (!s2iMode) {   
+         kubernetes.image().withName("${user}/${artifactId}:${config.version}").tag().inRepository("${env.FABRIC8_DOCKER_REGISTRY_SERVICE_HOST}:${env.FABRIC8_DOCKER_REGISTRY_SERVICE_PORT}/${user}/${artifactId}").withTag("${config.version}")
        }
     } else {
       if (!s2iMode) {        
         retry(3){
-          sh "mvn io.fabric8:fabric8-maven-plugin:${fabric8MavenPluginVersion}:push -Ddocker.push.registry=${env.FABRIC8_DOCKER_REGISTRY_SERVICE_HOST}:${env.FABRIC8_DOCKER_REGISTRY_SERVICE_PORT}"
+          sh "mvn fabric8:push -Ddocker.push.registry=${env.FABRIC8_DOCKER_REGISTRY_SERVICE_HOST}:${env.FABRIC8_DOCKER_REGISTRY_SERVICE_PORT}"
         }
       }
     }

